src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -150,10 +182,119 @@ private async Task<string> GetFormattedDiff(RenderedCodeFile renderedCodeFile, R
                 {
                     stringBuilder.Append("- ").Append(line.Line.DisplayString).Append(Environment.NewLine);
                 }
-                //No need to include unchanged lines for now. We will enhance this in next revision to include context.
             }
             stringBuilder.Append("```");
-            return stringBuilder.ToString();
+        }
+
+        private async Task<ReviewModel> GetBaseLineReview(string Language, string packageName, PullRequestModel pullRequestModel)
+        {
+            // Get  previously cloned review for this pull request or automatically generated master review for package
+            ReviewModel review;
```

Comment:
Is there potential race conditions here that might cause issues? i.e. 2 PRs/pipelines that might cause the same review to be created?

Reply:
Both pipelines will ideally have different copies of base line reviews. Let me check again if we need additional handling to handle this.